src/lib.rs

```diff
@@ -57,12 +62,16 @@ pub fn run_program_with_custom_state(bin_path: &str, mut vm: VMState) -> (U256,
                     _add(&mut vm, &opcode);
                 }
                 Variant::Sub(_) => _sub(&mut vm, &opcode),
+                Variant::Jump(_) => _jump(&mut vm, &opcode),
                 Variant::Mul(_) => _mul(&mut vm, &opcode),
                 Variant::Div(_) => _div(&mut vm, &opcode),
-                Variant::Jump(_) => todo!(),
                 Variant::Context(_) => todo!(),
                 Variant::Shift(_) => todo!(),
-                Variant::Binop(_) => todo!(),
+                Variant::Binop(binop) => match binop {
+                    zkevm_opcode_defs::BinopOpcode::Xor => _xor(&mut vm, &opcode),
+                    zkevm_opcode_defs::BinopOpcode::And => _and(&mut vm, &opcode),
+                    zkevm_opcode_defs::BinopOpcode::Or => _or(&mut vm, &opcode),
+                },
```

Comment:
Instead of `zkevm_opcode_defs::BinopOpcode::...` import `zkevm_opcode_defs::BinopOpcode` and then just use `BinopOpcode::...` so that it is clearer